tests/terraform/runner/test_plan_runner.py

```diff
@@ -276,7 +276,7 @@ def test_plan_runner_with_empty_vpc_connection(self):
         self.assertEqual(report.get_exit_code({'soft_fail': False, 'soft_fail_checks': [], 'soft_fail_threshold': None, 'hard_fail_checks': [], 'hard_fail_threshold': None}), 1)
         self.assertEqual(report.get_exit_code({'soft_fail': True, 'soft_fail_checks': [], 'soft_fail_threshold': None, 'hard_fail_checks': [], 'hard_fail_threshold': None}), 0)
 
-        self.assertEqual(report.get_summary()["failed"], 106)
+        self.assertEqual(report.get_summary()["failed"], 105)
```

Comment:
The [TFPlan](https://github.com/bridgecrewio/checkov/blob/872ec4b4abab77668dfdaa3f738c2e7a6c63937a/tests/terraform/runner/resources/plan/tfplan.json#L92-L97) has a MySQL RDS instance using `db.t3.micro` that [does not support Performance Insights](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PerfInsights.Overview.Engines.html).